integration-web/src/main/resources/log4j2.xml

```diff
@@ -40,18 +43,30 @@
             </Policies>
         </RollingFile>
 
+        <SymphonyOnPremHttpsAppender name="OnPremHttpsAppender" url="${sys:cloudLogHarvesterURL}"
+                                     maxLogsPerBulkRequest="${sys:symphony.maxLogsPerBulkRequest}"
+                                     maxLogsPerBulkRequestFromSink="${sys:symphony.maxLogsPerBulkRequest}"
+                                     maxPayloadSize="${sys:symphony.maxPayloadSize}" flushingInterval="5"
+                                     sinkFlushingInterval="5"
+                                     sinkFileName="${sys:logs.basedir}/logs/integration." instanceId="${sys:symphony.instanceId}"
+                                     sessionProviderClassName="com.symphony.integration.logging.IntegrationBridgeSessionProvider">
+            <PatternLayout pattern="%d %-5p [%c] (%t) %X{X-Trace-Id} %m%n"/>
```

Comment:
Put something around the trace ID so it's not free-floating.  Maybe '[Trace ID: %X{X-Trace-Id}]'